flutter/README.md

```diff
@@ -102,7 +102,7 @@ runApp(
 This adds performance tracing for all `AssetBundle` usages, where the `AssetBundle` is accessed with `DefaultAssetBunlde.of(context)`.
 This includes all of Flutters internal access of `AssetBundle`s, like `Image.asset` for example.
 Tracing for `AssetBundle.loadStructuredData()` is currently disabled.
-It's hidden by the `enableStructureDataTracing` flag and considered experimental. Using it could lead to bugs. We recognize the irony.
+It's hidden by the `enableStructuredDataTracing` flag and considered experimental. Using it could lead to bugs. We recognize the irony.
```

Comment:
Is it still experimental even though it's enabled by default?

Reply:
I removed the experimental notes, and never got any complaints.

Reply:
We can remove it on v8 maybe, it's always nice having an opt-out for whatever reason, at least for a bit.